doc/iterations/2024.3/shapes/search.adoc

```diff
@@ -0,0 +1,110 @@
+= (L) Add initial support for search
+
+== Problem
+
+Currently the only way a user to find elements of interest inside in his documents is to manually look into the explorer.
+
+For example he can "Expand all" from the root of all his documents and then use 'Ctrl-F' to open the filter, but:
+
+* this does not scale well for large models;
+* this only allows basic filtering/search on the element's labels (the only thing known to the frontend).
+
+We should provide support for search.
+This first version will keep things simple with a default implementation in Sirius Web Sample Application.
```

Comment:
I'm pretty sure that we will need that in both versions of Sirius Web so if a sizable part of the code is Sirius Web specific, it can either affect the cost of the feature or a choice will have to be made on the supported runtime.